<ui version="4.0" >
 <class>LinksExplorer</class>
 <widget class="QMainWindow" name="LinksExplorer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ссылки из других объектоа системы</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QTabWidget" name="tabWidget_Prg" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Hypotesis" >
       <attribute name="title" >
        <string>Ссылки из объектов ТИП</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" >
        <item row="0" column="0" >
         <widget class="QSplitter" name="splitter_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QEidosTreeWidget" name="HEidosTreeWidget" >
           <column>
            <property name="text" >
             <string>1</string>
            </property>
           </column>
          </widget>
          <widget class="QTableView" name="tableView_Hyp" />
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Pragma" >
       <attribute name="title" >
        <string>Ссылки из объектов ЭКЗЕМПЛЯР</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QEidosTreeWidget" name="PEidosTreeWidget" >
           <column>
            <property name="text" >
             <string>1</string>
            </property>
           </column>
          </widget>
          <widget class="QTableView" name="tableView_Pragma" />
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu" >
    <property name="title" >
     <string>Файл</string>
    </property>
    <addaction name="action_close" />
   </widget>
   <addaction name="menu" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="action_close" />
  </widget>
  <action name="action_close" >
   <property name="icon" >
    <iconset resource="../images/resources.qrc" >
     <normaloff>:/PICS/exit.png</normaloff>:/PICS/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Закрыть форму</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEidosTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qeidostreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>action_close</sender>
   <signal>activated()</signal>
   <receiver>LinksExplorer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>502</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
